app/models/miq_worker/runner.rb

```diff
@@ -489,4 +494,13 @@ def process_title
   def set_process_title
     Process.setproctitle(process_title)
   end
+
```

Comment:
Sorry, I'm being nitpicky...

* `def skip_heartbeat?` with env var logic in one spot?
* `private` these two methods (lolz the protected on 461)

`private` (lolz the protected on 461)